std/digest/package.d

```diff
@@ -123,7 +123,7 @@ import std.traits;
     {
         File file = File(filename);
 
-        //As digests imlement OutputRange, we could use std.algorithm.copy
+        //As digests imlement OutputRange, we could use std.range.put
```

Comment:
Can we revert this? `copy` should work for this now. Alternatively, the comment looks odd since you are calling `hash.put`, which I first thought was a UFCS call of `std.range.put`. If we leave the comment as calling `std.range.put`, maybe include the full call to show how it's different.